src/routes/Private.jsx

```diff
@@ -98,17 +99,19 @@ class Private extends React.Component {
                       classes={{ selected: classes.tabSelected }}
                     />
                   </Tabs>
-                  <Switch className={classes.component}>
-                    <Route exact path="/" component={HomeView} />
-                    {features.searchui && (
-                      <Route exact path="/explorer" component={DataExplorerView} />
-                    )}
-                    <Route exact path="/datasets" component={DatasetsView} />
-                    <Route exact path="/datasets/:uuid/query" component={QueryView} />
-                    <Route exact path="/datasets/:uuid/details" component={DatasetDetailView} />
-                    <Route exact path="/snapshots" component={SnapshotView} />
-                    <Route exact path="/snapshots/details/:uuid" component={SnapshotDetailView} />
-                  </Switch>
+                  <div className={classes.component}>
```

Comment:
More CSS in the routing layer? We sure we need this?

Reply:
yeah, this component class was previously being applied to the `Switch` which doesn't actually work, but we want the overflow property to be set here instead of doing it on every new component

Reply:
Ok, probably at some point we should have a wrapping component wraps all pages for these kinds of things